### DISCUSSION TITLE
Running both Paperless and Vaultwarden LXCs

### DISCUSSION THREAD

**Discussion by altanir84** [tteck/Proxmox#2153] [Help Needed]
Hello. I'd like to ask some help on how to run both of those applications at the same time. Both use Interface: IP:8000. How can I change the port for one of them?

Thanks very much!

**Comment by tteck** [comment]
No issue if they have different IPs.

**Reply by altanir84** [reply]
yeah, I now realize it was a dumb question. Thanks for the quick reply!
